### PR TITLE
V1: Rollbar.NET notifier as Asp.Net Core 2 middleware 

### PR DESCRIPTION
**Pull request by akornich**

### CODE REVIEW

File: SolutionCommon.csproj [hunk 1]
```diff
@@ -21,5 +21,15 @@
   <PropertyGroup>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Rollbar.xml</DocumentationFile>
   </PropertyGroup>
-  
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0'">
+    <DefineConstants>NETCOREAPP2_0</DefineConstants>
+    <!---DefineConstants>NETCOREAPP2_0;NETSTANDARD2_0</DefineConstants-->
```
Comment (rokob) [inline SolutionCommon.csproj]: We can get rid of this comment right?

File: Rollbar/AspNetCore/RollbarMiddleware.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                // so we are trying to configure it from IConfiguration:
+
+                const string defaultAccessToken = "none";
+                RollbarConfig rollbarConfig = new RollbarConfig(defaultAccessToken);
+                configuration.GetSection("Rollbar").Bind(rollbarConfig);
+
+                if (rollbarConfig.AccessToken == defaultAccessToken)
+                {
+                    const string error = "Rollbar.NET notifier is not configured properly.";
+                    //Debug.Assert(false, error);
```
Comment (rokob) [inline Rollbar/AspNetCore/RollbarMiddleware.cs]: We can get rid of this

File: SolutionCommon.csproj (hunk 1 above)
Comment (akornich) [reply SolutionCommon.csproj]: sure. gone.

File: Rollbar/AspNetCore/RollbarMiddleware.cs (hunk 2 above)
Comment (akornich) [reply Rollbar/AspNetCore/RollbarMiddleware.cs]: gone.

**Pull request merged**